ktor-client/ktor-client-apache5/jvm/src/io/ktor/client/engine/apache5/Apache5Engine.kt

```diff
@@ -89,6 +93,8 @@ internal class Apache5Engine(override val config: Apache5EngineConfig) : HttpCli
                             ClientTlsStrategyBuilder.create()
                                 .setSslContext(config.sslContext ?: SSLContexts.createSystemDefault())
                                 .setTlsVersions(TLS.V_1_3, TLS.V_1_2)
+                                // TODO: Remove apply after https://github.com/apache/httpcomponents-client/pull/588 merged
+                                .apply { setHostnameVerificationPolicy(HostnameVerificationPolicy.BOTH) }
```

Comment:
Should we make this value [configurable](https://javadoc.io/doc/org.apache.httpcomponents.client5/httpclient5/latest/org/apache/hc/client5/http/ssl/HostnameVerificationPolicy.html)?

Reply:
yep, it would be nice

Reply:
Done

Reply:
Could you check docs for the added field?